doc/source/ray-core/scheduling/configure-memory-monitor.rst

```diff
@@ -0,0 +1,39 @@
+Configure memory monitor
+========================
+
+The memory monitor periodically checks the memory usage of the node and reports whether 
```

Comment:
What about we go from the high level problem first -> explain the tool? 

Suggestion: in Linux, high RAM usage can kill workers or raylets randomly which breaks Ray -> we have memory monitor to prevent this -> explain what this does -> whether or not it is on/off and how to configure (link)